manager/build.gradle.kts

```diff
@@ -89,6 +89,24 @@ fun Project.configureBaseExtension() {
             signingConfigs["debug"]
         }
 
+        if (signFile.canRead()) {
+            val prop = Properties()
+            prop.load(signFile.inputStream())
+            var APPLICATION_ID = prop.getProperty("APPLICATION_ID")
+            var APPLICATION_NAME = prop.getProperty("APPLICATION_ID")
+            if (APPLICATION_ID.isBlank()){
+                APPLICATION_ID = "me.weishu.kernelsu"
```

Comment:
better to use a variable such as `DEFAULT_APPLICATION_ID`, because it is used later again.